_data/hosting_providers.json

```
@@ -2278,5 +2278,15 @@
     "plan": "",
     "reviewed": "2020.01.07",
     "note": "Available for all plans from free account to dedicated servers"
+  },
+  {
+    "name": "Wikstrom Telephone Company (Wiktel)",
+    "link": "https://wiktel.com",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "",
+    "plan": "https://wiktel.com/webhosting",
```

This isn't quite what this field is for.
```suggestion
    "plan": "",
```